demo/midgard-node/src/config.ts

```diff
@@ -98,6 +99,7 @@ export const makeConfig = Effect.gen(function* () {
     Config.string("MEMPOOL_MPT_DB").pipe(
       Config.withDefault("./midgard-mempool-mpt-db"),
     ),
+    Config.string("GENESIS_UTXOS_PATH").pipe(Config.withDefault(null)),
```

Comment:
Can you please add a default value here as well that aligns with the current structure?